### PR TITLE
fix: Fix logger, Yarn 2 and components structure

### PR DESCRIPTION
**Pull request by maciejopalinski**

### CODE REVIEW

**Review by Mariusz-web — APPROVED** [review]

File: src/utils/Logger.ts [hunk 1]
```diff
@@ -1,22 +1,15 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
 export class Logger {
-    static info(tag: string, message: string, context?: any) {
-        if (process.env.REACT_APP_DEVELOPER_MODE !== 'true') {
-            return;
-        }
-        console.log(`[${tag}] : ${message}`, { context });
+    static debug(...params: unknown[]) {
```
Comment (Veryyapeee) [inline src/utils/Logger.ts]: Don't change that file at all

File: src/components/Header.tsx [hunk 2]
```diff
@@ -7,10 +7,10 @@ import Toolbar from '@mui/material/Toolbar';
 import Typography from '@mui/material/Typography';
 import React, { useState } from 'react';
 
-import { Navigation } from '../Atoms/Navigation';
+import { Navigation } from '.';
 
 export const Header = () => {
-    const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+    const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
```
Comment (Veryyapeee) [inline src/components/Header.tsx]: unnecessary changes

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/utils/Logger.ts (hunk 1 above)
Comment (maciejopalinski) [reply src/utils/Logger.ts]: but that's a simpler way of logging, just like console.log works... why complicate stuff that doesn't need to be complicated

Comment (Veryyapeee) [reply src/utils/Logger.ts]: For standards which is very important for logs, this task has nothing to do with that, recept that changes please

File: src/utils/Logger.ts [hunk 3]
```diff
@@ -1,22 +1,16 @@
...
+    static debug(tag: string, message: string, context?: unknown) {
+        if (process.env.REACT_APP_DEBUG !== 'true') {
             return;
         }
-        console.log(`[${tag}] : ${message}`, { context });
+        console.debug(`[${tag}] : ${message}`, { context });
     }
 
-    static debug(tag: string, message: string, context?: any) {
-        if (process.env.REACT_APP_DEVELOPER_MODE !== 'true') {
```
Comment (Veryyapeee) [inline src/utils/Logger.ts:11]: Why are you removing these ifs

We dont want any logs in production

I dont know why are you even making any changes in this file in scope od this task

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/utils/Logger.ts (hunk 3 above)
Comment (maciejopalinski) [reply src/utils/Logger.ts:11]: we don't want errors? yeah ok whatever
and it is not a task, this is a bugfix

**Review by Veryyapeee — APPROVED** [review]
